src/vignette/storage/static_assets.clj

```diff
@@ -24,7 +24,7 @@
 
 (defn create-async-response-stored-object [static-image-get image-review-get]
   (let [static-image-response @static-image-get]
-    (if (and (-> static-image-response :status (= 200)) (some #(= % (:status @image-review-get)) [200 404]))
+    (if (and (-> static-image-response :status (= 200)) (some #(not (= % (:status @image-review-get))) [403]))
```

Comment:
This seems like it's fundamentally changing what this function is doing. Why did it used to check for a `200` or `404` but now it needs to check for not a `403`? Is the other check no longer required?